api/Navigator.json

```diff
@@ -2459,6 +2459,62 @@
             "standard_track": true,
             "deprecated": true
           }
+        },
+        "returns_plugin_type": {
```

Comment:
This is the "inverted" subfeature indicating when `Navigator.mimeTypes` returns mime types from installed plugins. Following the version_removed it returns hard coded values used to mock the `Navigator.pdfViewerEnabled` property.